Tasks/NuGetCommandV2/nugetcommandmain.ts

```diff
@@ -30,6 +31,18 @@ async function main(): Promise<void> {
         if (nugetVersionInfo && nugetVersionInfo.fileVersion){
             nugetVersion = nugetVersionInfo.fileVersion.toString();
         }
+
+        const nugetSemVer = semver.coerce(nugetVersion);
+        if (semver.lt(nugetSemVer, '4.8.0'))
+        {
+            const msbuildSemVer = await nuGetGetter.getMSBuildVersion();
+            if (semver.gte(msbuildSemVer, '16.5.0'))
```

Comment:
We already call getNuGet() earlier in this file and getNuGet() also has this warning so I think it's sufficient to only have it there.

Reply:
Yeah the earlier call sometimes short circuits before we get to the code path we need.